internal/upload/koji/run-koji-container.sh

```diff
@@ -46,7 +46,7 @@ ${CONTAINER_RUNTIME} run -d --name org.osbuild.koji.koji --network org.osbuild.k
   -e POSTGRES_PASSWORD=kojipass \
   -e POSTGRES_DB=koji \
   -e POSTGRES_HOST=org.osbuild.koji.postgres \
-  quay.io/osbuild/ghci-koji:rc1
+  quay.io/osbuild/ghci-koji:v1
```

Comment:
Intentional, or did we do something wrong? @dvdhrm?

Reply:
That `rc1` doesn't exist any more, I mean.

Reply:
No idea. The primary issue is that I left there `rc1` for many months, I should have changed it to `v1` but forgot.